README.md

```diff
@@ -272,6 +275,17 @@ It may be useful to have some <tt>GRANT</tt> commands or similar run here when l
 table, the changes will be reverted and the backup table restored if post actions fail.</p>
     </td>
  </tr>
+ <tr>
+    <td><tt>avrocompression</tt></td>
+    <td>No</td>
+    <td>No compression (unless set in Hadoop config)</td>
+    <td>
+<p>Sets the compression codec to use on the Avro data to be loaded into Redshift. This overwrites the <tt>avro.output.codec</tt>
```

Comment:
We should probably verify that Redshift supports all of these compression modes. I don't see why it would necessarily support fewer of them, but it's possible. I may be able to take care of this as part of integration testing (I might add a new suite which tests all of the compression options).